### PR TITLE
Added management api specification

### PR DESCRIPTION
**Pull request by jigar-arc10**
I have added the initial management API specification. 
We will make changes in terms of granular specifications on the APIs.

### CODE REVIEW

**Comment by anilmurty** [comment]
Thanks for getting this out Jigar. It's a great first version. Here are my comments:
1. Ideally we should document the request endpoints here as well (e.g api/v1/provider/LeaseStatus or api/v1/provider/{gseq}/getLease)
2. I think it might help to have an endpoint to request ALL leases (no DSEQ specified) (e.g api/v1/provider/leaseList or something)
3. I think we may want to also return the image names (in case we intend to implement a allow/ deny list in the moderation APIs)?

**Comment by anilmurty** [comment]
Notes from the wg meeting: Jigar and Deval working on an update to this and will post an update to this PR

**Comment by jigar-arc10** [comment]
Discussed with Deval and reviewed existing APIs to form new management API standards.

File: wg-content-moderation/management-api.md [hunk 1]
```diff
@@ -6,8 +6,65 @@ Provider tooling support
...
+|`deployment.state` | Deployment current state |
+|`deployment.urls` | if deployed on port 80 return urls |
+|`deployment.ports` | if deployed on port other than 80 or 443, return host and provider ports |
+|`owner` | lease owner wallet address |
+|`lease.amount` | Lease price per block |
+
+Response will contain above reponse fields for lease
+
+### Close Lease
+#### `/api/v1/provider/lease/close/?dseq=&gseq= (Lease List)`
```
Comment (boz) [inline wg-content-moderation/management-api.md:69]: this (and all things that reference a deployment) should include the tenant address.

A deployment ID is `(tenant-address,dseq)`.

**Review by boz — COMMENTED** [review]
comment re IDs.

File: wg-content-moderation/management-api.md (hunk 1 above)
Comment (anilmurty) [reply wg-content-moderation/management-api.md:69]: Per discussion during the wg call: Agreed to include these in the lease ID

**Review by anilmurty — APPROVED** [review]
Approved by all

**Pull request merged**